src/collective/nitf/upgrades/v2000/__init__.py

```diff
@@ -44,6 +44,16 @@ def update_layouts(context, logger=None):
     for item in results:
         obj = item.getObject()
         if obj.getLayout() == 'nitf_galleria':
-            obj.setLayout('galleria')
+            obj.setLayout('slideshow')
             i += 1
         logger.info(u'{0} News Articles updated'.format(i))
+
+
+def install_new_dependencies(context, logger=None):
+    logger = logging.getLogger(PROJECTNAME)
+    dependencies = ('collective.js.cycle2',)
+    qi = api.portal.get_tool('portal_quickinstaller')
+    for p in dependencies:
+        if not qi.isProductInstalled(p):
+            qi.installProducts([p])
+            logger.info(u'{0} installed'.format(p))
```

Comment:
no, as you don't know if some other package is using it; we need to document that.